.eslintrc.json

```diff
@@ -1,11 +1,6 @@
 {
-  "plugins": [
-    "jest"
-  ],
-  "extends": [
-    "airbnb",
-    "plugin:jest/recommended"
-  ],
+  "plugins": ["jest"],
```

Comment:
I feel like these aren't the actual linter rules. I'm wondering if you have different settings enabled for yours? Can we just exclude this file from the history here?

Reply:
Actually, someone mentioned that you might be using something like beautify, as these changes don't look like some of our linter rules. I have a feeling when you merge in the latest master, you're going to get blocked from submitting these changes. Are you using any extra extensions that may be doing this besides the ones that I recommend when we initially installed the linter rules?

Reply:
I am using Prettier.  Isn't this the purpose of the ESLint file, to give rules for all these formatting issues?

Reply:
I think, as a team, we should be consistently using the same styling/linting tools. Neither of us (I believe Jack doesn't) use prettier. That is something separate from the eslinter tools we have set up. Ideally, we want to get rid of all of these prettier changes that are overcomplicating this PR.

Here are some next steps:
1) Undo the linter changes in files that are _just_ in the diff, as I think most of this is due to prettier.
2) Merge in the latest master.
3) Checkout the README file. I believe I added steps to make sure your local env is setup to follow the ESLint rules we have. Let's make sure you have everything set up properly. DM me if you have any questions about the setup.
4) Turn prettier off for this project.

That should get you in a good place to be consistent with the conventions that Jack and I are following. Also, it'll help clear up the diff in this PR.